l10n_br_fiscal/models/document_fiscal_line_mixin_methods.py

```diff
@@ -810,3 +854,11 @@ def _onchange_fiscal_tax_ids(self):
     def _onchange_city_taxation_code_id(self):
         if self.city_taxation_code_id:
             self.cnae_id = self.city_taxation_code_id.cnae_id
+
+    @api.model
+    def _add_fields_to_amount(self):
+        return ['insurance_value', 'other_value', 'freight_value']
+
+    @api.model
+    def _rm_fields_to_amount(self):
+        return ['discount_value', 'icms_relief_value']
```

Comment:
It shouldn't return the field `discount_value` because this field is already discounted in the calculation of amount_untaxed: https://github.com/OCA/l10n-brazil/blob/22a3500625ba917b7e5a54dd2abbe018d2dd8cba/l10n_br_fiscal/models/document_fiscal_line_mixin_methods.py#L148